scripts/library_handler.py

```diff
@@ -146,18 +146,16 @@ def checkSameVersion(expected, received):
   #Only check as many digits as given in expSplit
   rcvSplit = [int(x) for x in received.split('.')[:len(expSplit)]]
   # drop trailing 0s on both
-  while expSplit[-1] == 0:
+  while len(expSplit) and expSplit[-1] == 0:
```

Comment:
This is needed now since xarray is at version 2023 (not following the standard nomenclature of "major.minor.patch")
Without this fix, the library handler crashes if a user has a version of xarray =/ 2023